servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/ScopeTracker.java

```diff
@@ -74,23 +76,12 @@ public void onError(final Throwable throwable) {
     @Override
     public void cancel() {
         try {
-            tagStatusCode();
-            span.setStatus(StatusCode.ERROR);
+            instrumenter.end(context, request, metaData, null);
```

Comment:
I wonder if this case should report with an exception because passing `null` indicates "success", but cancel doesn't mean success in practice.
Consider passing a static instance of `new Exception("cancelled", null, false, false)` here and then check by reference inside `ServicetalkSpanStatusExtractor`.

Reply:
ok, I created an exception but the exception is enough as the ```ServicetalkSpanStatusExtractor ``` will mark the span as error.